4week/src/hooks/queries/useThrottle.ts

```
@@ -0,0 +1,55 @@
+import { useRef, useCallback, useEffect } from "react";
+
+/**
+ * @param callback 쓰로틀을 적용할 함수
+ * @param delay 쓰로틀 지연 시간 (ms)
+ * @returns 쓰로틀이 적용된 함수
+ */
+export const useThrottle = (
+  callback: (...args: any[]) => void,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)